.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name:  Build protobuf stubs
+
+on:
+  push:
```

Comment:
If left on all branches then it will regenerate in work branches which I don't think is ideal.